edit/BUILD.bazel

```diff
@@ -34,3 +35,14 @@ go_test(
     embed = [":go_default_library"],
     deps = ["//build:go_default_library"],
 )
+
+go_library(
+    name = "safebuildifier",
```

Comment:
Having multiple go-libraries works well with Bazel, however not compatible if one uses standard go tooling (e.g. `go build`). Could you please move `safebuildifier.go` and this `go_library` definition to another directory (can be a subdirectory, e.g. `edit/safe`?

Reply:
Done.